Lib/test/test_msvcrt.py

```diff
@@ -146,18 +151,30 @@ def test_CrtSetReportMode(self):
                                            msvcrt.CRTDBG_MODE_DEBUG)
         self.assertIs(type(returned), int)
         self.assertNotEqual(returned, -1)
```

Comment:
It is equal to msvcrt.CRTDBG_REPORT_MODE

Reply:
Same as comments in `set_error_mode`, and updated.